visual-retrieval-colpali/frontend/layout.py

```diff
@@ -107,6 +107,11 @@ def ThemeToggle(variant="ghost", cls=None, **kwargs):
 
 def Links():
     return Nav(
+        A(
+            Button("what is this?", variant="link"),
```

Comment:
```suggestion
            Button("What's this?", variant="link"),
```
imho, this looks a tiny bit better?